Sources/Views/MessageLabel.swift

```diff
@@ -462,4 +524,46 @@ open class MessageLabel: UILabel, UIGestureRecognizerDelegate {
         delegate?.didSelectPhoneNumber(phoneNumber)
     }
     
+    // MARK: UIGestureRecognizer Delegate
+    
+    public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
+        return true
+    }
+    
+    //swiftlint:disable cyclomatic_complexity
+    // Yeah we're disabling this because the whole file is a mess :D
+    public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldReceive touch: UITouch) -> Bool {
+        
```

Comment:
Gestures are broken because logic is so brittle. Would be good to find a better solution this time around 😓

Reply:
How about rather than using gestures use the `touchesBegan` function thats in every `UIView`

Reply:
I can look into it but the good thing about gestures are:
1. long press gestures
2. they are cancellable (which we need)